biosimulations/apps/simulators/src/app/help/faq/faq.component.html

```diff
@@ -82,59 +84,15 @@
       tocSection="Simulation outputs"
     >
       <biosimulations-q-a heading="Which formats does BioSimulators support?">
-        Currently, reports are saved in Comma-Separated Values (CSV) format and plots are saved in Portable Document Format (PDF).
```

Comment:
Are we looking to do this before the 20th?

Reply:
No, for now lets keep our goal of runBioSimulations using CSV. For now, lets focus on the singularity containers that Akhil manually created.

I'm moving this discussion to #1669.